inst/include/common/def.hpp

```diff
@@ -15,7 +15,18 @@
 #include <memory>
 #include <vector>
 
-std::ofstream FIMS_LOG("fims.log"); /**< Log file */
+std::ofstream FIMS_LOG ("logs/fims.log"); /**< Log file */
```

Comment:
add inline documentation (using //) here describing what these files are

Reply:
Let me know if this seems sufficient.